docs/CodeLens.md

```diff
@@ -54,3 +54,9 @@ Here is a demonstration of the above Code Lens:
 Here is a demonstration of the code lens for `xs:element`:
 
 ![An element root is declared that contains an element boot. Above the declaration of boot, there is a code lens that links to the reference in root.](./images/CodeLens/XSDElementCodeLens.gif)
+
+### XML Features
```

Comment:
I think that it might be a good idea to put this at the top of the page, since it's the feature that user's are most likely to encounter